docsets/Tailwind_CSS/docset.json

```diff
@@ -13,7 +13,7 @@
   ],
   "specific_versions": [
     {
-      "version": "2.0.x/14_2020-11-22",
+      "version": "2.1.x/15_2021-04-07",
       "archive": "versions/2.0.x/tailwindcss.tgz"
```

Comment:
You need to change the path to `versions/2.1.x/tailwindcss.tgz` I think.

Reply:
wth. and i'm not even drunk.